ui/src/routes/admin/users/index.tsx

```diff
@@ -152,14 +152,13 @@ const Users = () => {
             <TooltipTrigger asChild>
               <Button asChild size='sm' className='ml-auto gap-1'>
                 <Link to='/admin/users/create-user'>
-                  Add user
+                  Create new user
                   <UserPlus className='h-4 w-4' />
                 </Link>
               </Button>
             </TooltipTrigger>
             <TooltipContent>
-              Add a new user to the server. Note that the username has to be
-              unique.
+              Create a new user account for the server.
```

Comment:
In the tooltip, I'm fine to be more verbose and say "new", but I'd omit "for the server".

But if you're omitting "Note that the username has to be unique." already in this commit, it would be nice to also see where it's being moved to already in this commit. So from a reviewer's perspective no information is lost. Or, if too complicated to achieve, announce in the commit message to restore that hint later.